VideoWeb/VideoWeb.AcceptanceTests/Steps/DataSetupSteps.cs

```diff
@@ -115,6 +115,8 @@ public void GivenIHaveAHearing(int minutes = 0, string location = "Birmingham Ci
                 .WithScheduledTime(_c.TimeZone.AdjustAnyOS(DateTime.Now.ToUniversalTime().AddMinutes(minutes)))
                 .WithScheduledDuration(HearingDuration)
                 .WithLocation(location)
+                .WithAnObserver()
+                .WithAPanelMember()
                 .Build();
```

Comment:
can you take this out? There is already a method to add an observer and panel member, and this will mean ALL tests have them which will break some

Reply:
Given I have a hearing with an Observer and Panel Member

Reply:
removed the observer and panel member from the Createhearing